From 238521506fef43a7ccfeba60ec4e06895207d43f Mon Sep 17 00:00:00 2001
From: Kei Okada <k-okada@jsk.t.u-tokyo.ac.jp>
Date: Fri, 25 Apr 2014 16:47:38 +0900
Subject: [PATCH] remove unsupported packages for groovy

---
 CMakeLists.txt |    6 +++---
 package.xml    |    6 +++---
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e473813..3be3e7a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -17,10 +17,10 @@ if (${roseus_msgs_generate})
     moveit_setup_assistant
     pr2_moveit_plugins
     # pr2eus
-    pr2_msgs
-    pr2_controllers_msgs
+    # pr2_msgs
+    # pr2_controllers_msgs
     control_msgs
-    move_base_msgs
+    # move_base_msgs
     nav_msgs
     dynamic_reconfigure
 
diff --git a/package.xml b/package.xml
index 8692916..143e8f8 100644
--- a/package.xml
+++ b/package.xml
@@ -14,10 +14,10 @@
 
   <build_depend>rviz</build_depend>
   <build_depend>moveit_full_pr2</build_depend>
-  <build_depend>pr2_msgs</build_depend>
-  <build_depend>pr2_controllers_msgs</build_depend>
+  <!-- <build_depend>pr2_msgs</build_depend> -->
+  <!-- <build_depend>pr2_controllers_msgs</build_depend> -->
   <build_depend>control_msgs</build_depend>
-  <build_depend>move_base_msgs</build_depend>
+  <!-- <build_depend>move_base_msgs</build_depend> -->
   <build_depend>nav_msgs</build_depend>
   <build_depend>dynamic_reconfigure</build_depend>
 
-- 
1.7.9.5

