From 33c4b94fbf168225df2e1acd8435c57cb792c15d Mon Sep 17 00:00:00 2001
From: Kei Okada <k-okada@jsk.t.u-tokyo.ac.jp>
Date: Sun, 11 May 2014 16:28:17 +0900
Subject: [PATCH 1/2] add roseus-msgs to Depends

---
 debian/control.em |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control.em b/debian/control.em
index 8b9cd67..a6b5157 100644
--- a/debian/control.em
+++ b/debian/control.em
@@ -8,5 +8,5 @@ Standards-Version: 3.9.2
 
 Package: @(Package)
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends))
+Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends)), ros-hydro-roseus-msgs
 Description: @(Description)
-- 
1.7.9.5

