From 66963bc891a0bcec8dc43de33642c8828ee9ffcf Mon Sep 17 00:00:00 2001
From: Kei Okada <k-okada@jsk.t.u-tokyo.ac.jp>
Date: Sun, 27 Apr 2014 16:21:06 +0900
Subject: [PATCH] add deb for ros build firm

---
 debian/control.em |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/control.em b/debian/control.em
index 35e0893..3b7144d 100644
--- a/debian/control.em
+++ b/debian/control.em
@@ -2,11 +2,11 @@ Source: @(Package)
 Section: misc
 Priority: extra
 Maintainer: @(Maintainer)
-Build-Depends: debhelper (>= 7.0.50~), @(', '.join(BuildDepends))
+Build-Depends: debhelper (>= 7.0.50~), @(', '.join(BuildDepends)), ros-groovy-pr2-common, ros-groovy-pr2-controllers, ros-groovy-navigation
 Homepage: @(Homepage)
 Standards-Version: 3.9.2
 
 Package: @(Package)
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends))
+Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends)), ros-groovy-pr2-common, ros-groovy-pr2-controllers, ros-groovy-navigation 
 Description: @(Description)
-- 
1.7.9.5

