From 28b78977e06d061f087f2625aef96258e0d1ab1b Mon Sep 17 00:00:00 2001
From: Kei Okada <k-okada@jsk.t.u-tokyo.ac.jp>
Date: Mon, 12 May 2014 23:11:50 +0900
Subject: [PATCH 2/2] disable to install ros-indigo-roseus-msgs

---
 debian/control.em |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control.em b/debian/control.em
index 6ca9492..8b9cd67 100644
--- a/debian/control.em
+++ b/debian/control.em
@@ -8,5 +8,5 @@ Standards-Version: 3.9.2
 
 Package: @(Package)
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends)), ros-indigo-roseus-msgs
+Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends))
 Description: @(Description)
-- 
1.7.9.5

